install_config/aggregate_logging.adoc

```diff
@@ -676,6 +675,11 @@ This loopback must be maintained manually outside of {product-title}, on the
 node. You must not maintain it from inside a container.
 ====
 
+You can use Container-Native Storage (CNS) 3.6 to enable Gluster Block Storage
```

Comment:
A few things:
* There is no such thing as CNS in the upstream. If possible, please see if you can make use of the variables I'm introducing in #8149, namely `{gluster-native}`, as that will correctly present wither "Containerized GlusterFS" or "Container-Native Storage" depending on the doc site.
* In my mind there's no need to mention a specific version. If you must, I request that you instead say "3.6 and onward" and that you encapsulate it in an `ifdef::openshift-enterprise[]` block.
* I've been stylizing it as `gluster-block` in my documentation. I think that's correct? If there's any official stylization for it I'd be happy to know and correct my docs if needed.